hazelcast/src/test/resources/test-hazelcast-real-jcache.xml

```diff
@@ -0,0 +1,80 @@
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<hazelcast xsi:schemaLocation="http://www.hazelcast.com/schema/config hazelcast-config-3.5.xsd"
+           xmlns="http://www.hazelcast.com/schema/config"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+
+    <group>
+        <name>test-group1</name>
+        <password>test-pass1</password>
+    </group>
+
+    <network>
+        <join>
+            <tcp-ip enabled="true">
+                <member>127.0.0.1</member>
+            </tcp-ip>
+        </join>
+    </network>
+
+    <cache name="cache1">
+        <key-type class-name="java.lang.Object"/>
+        <value-type class-name="java.lang.Object"/>
+        <statistics-enabled>true</statistics-enabled>
+        <management-enabled>true</management-enabled>
+        <read-through>true</read-through>
+        <write-through>true</write-through>
+        <cache-loader-factory class-name="com.hazelcast.cache.CacheConfigTest$MyCacheLoaderFactory"/>
+        <cache-writer-factory class-name="com.hazelcast.cache.CacheConfigTest$MyCacheWriterFactory"/>
+        <expiry-policy-factory class-name="com.hazelcast.cache.CacheConfigTest$MyExpirePolicyFactory"/>
+        <eviction size="50" max-size-policy="ENTRY_COUNT" eviction-policy="LRU"/>
+        <cache-entry-listeners>
+            <cache-entry-listener old-value-required="false" synchronous="false">
+                <cache-entry-listener-factory class-name="com.hazelcast.cache.CacheConfigTest$MyEntryListenerFactory"/>
+                <cache-entry-event-filter-factory class-name="com.hazelcast.cache.CacheConfigTest$MyEntryEventFilterFactory"/>
+            </cache-entry-listener>
+           <cache-entry-listener old-value-required="true" synchronous="true">
+                <cache-entry-listener-factory class-name="com.hazelcast.cache.CacheConfigTest$MySyncEntryListenerFactory"/>
+                <cache-entry-event-filter-factory class-name="com.hazelcast.cache.CacheConfigTest$MySyncEntryEventFilterFactory"/>
+            </cache-entry-listener>
+        </cache-entry-listeners>
+        <backup-count>3</backup-count>
+        <async-backup-count>2</async-backup-count>
+    </cache>
+
+    <cache name="testCache">
+        <key-type class-name="java.lang.Integer"/>
+        <value-type class-name="java.lang.String"/>
+        <statistics-enabled>false</statistics-enabled>
+        <management-enabled>false</management-enabled>
+    </cache>
+
+    <cache name="wanRefTestCache">
```

Comment:
I think, `wanRefTestCache` cache config is not needed if this xml configuration file is not used by other tests.